tests/unit/requirements.txt

```diff
@@ -1,3 +1,5 @@
 # For cos_agent library
 cosl
 pydantic < 2
+pytest
+pytest-mock
```

Comment:
Can we use mocking from unittests? We are not using any feature of this library, so I would prefer to avoid adding it as decency.

Reply:
I tend to not mix the unittest and pytest. Let's keep it this style?

Reply:
What mixing, importing patch and Mock, MagicMock directly is not mixing. This is definitely not blocking comment.

Reply:
Since it's not a blocking comment, I would like to resolve this one and keep what we have right now.